### ISSUE TITLE
what's the specific parameter？

### ISSUE THREAD

**Issue by pht6** [Zood123/MLLM-Unlearning-Attack#1]
"Could you please specify the hyperparameters used for training the vanilla model and the noise? Specifically, I am interested in the number of epochs, the learning rate schedule (e.g., constant, linear decay, or cosine annealing), and the batch size."

"Once I successfully reproduce the results, I will make sure to cite your work in my upcoming publications."

thanks very much

**Comment by Zood123** [comment]
Hi, thank you for your interest. 
I set the number of iterations as 800. Alpha as 1. batch_size as 6.
Also, the unlearned NPO is at hugging face: xzz5508/SUA_NPO_Unlearned
Data to attack: xzz5508/SUA_MLLMU

**Comment by pht6** [comment]
thank for your review，and what about the parameter to train the vanilla model and unlearned model（grad difference）？

**Comment by Zood123** [comment]
Vanilla Model:
Batch size: 4
lr: 4e-5

grad difference:
Batch size: 3
lr: 1e-5
